deploy/k8s/manifest.yaml

```diff
@@ -277,6 +277,10 @@ data:
             experimental_match_labels: { "condition": "DiskPressure", "status": "true" }
             action: insert
             new_name: k8s.node.status.condition.diskpressure
+          - include: k8s.kube_pod_status_phase
+            experimental_match_labels: { "sw.k8s.pod.status": "Running" }
+            action: insert
+            new_name: k8s.pod.status.phase.running
```

Comment:
Do we use this new metric anywhere? It's not removed after used to create `k8s.cluster.pods.running`.